.gitlab/configs/common.yml

```diff
@@ -9,6 +9,10 @@
 # terms of the BSD-3 license. We welcome feedback and contributions, see file
 # CONTRIBUTING.md for details.
 
+include:
+  - project: 'lc-templates/id_tokens'
+    file: 'id_tokens.yml'
+
```

Comment:
Does this need to be here as well? The LC documentation says it has to be added to `.gitlab-ci.yml` but does not mention other files.

Reply:
What this does under the hood is adding a default configuration for every job of every pipelines. LC legitimately asks to place this in the .gitlab-ci.yml, but since we have child pipelines, we also need to make sure to apply the change for each of them too.
So, yes, it needs to be here as well.